### DISCUSSION TITLE
What's the proper way to run command for single package that will trigger the same command for all dependent pacakges

### DISCUSSION THREAD

**Discussion by desfero** [nrwl/nx#7597] [General]
I'm not able to find any example to fit the need i think is quite common.

Given that we have 2 libs that our `web` app depends on i want to run single build for the `web` app that will under the hood build dependent libraries (in the proper order when needed)

Tried with the following commands:
```
nx run web:build
nx run-many --target build --projects=web --with-deps
```
Is there something wrong with my nx setup or the given use-case is not supported?

**Comment by brandonroberts** [comment]
What version of Nx are you on? If your libraries are buildable, this happens by default with the `targetDependencies` defined in the `nx.json` file. If your libraries are not buildable, they are just compiled from source as part of the `web` build.

**Reply by desfero** [reply]
I'm on the lates version. 
Seems i see the case of my small confusion:
```
nx run-many --target=start --projects=web --parallel
```
the above command is not working in the way i just expected it to work. 

Given that `start` command blocks the console it won't start any watchers except the first one.
For e.g let's assume we have the following deps structure `shared-utils` -> `shared-modules` -> `web` it will just start the first `shared-utils` watcher and block all next commands to run even when `--parallel` is provided.

Is there then some way to `start` all `web` deps and `web` app through NX without manually specific each projects 
```
nx run-many --target=start --projects=shared-utils,shared-modules,web --parallel
```
and removing `start` from `targetDependencies` config
